osprey/processes/wps_convolution.py

```diff
@@ -29,6 +30,56 @@ def log_handler(process, response, message, process_step=None, level="INFO"):
 
 class Convolution(Process):
     def __init__(self):
+        self.config_dict = {
+            # configuration dictionary used for RVIC convolution
+            # required user inputs are defined as None value
+            "OPTIONS": {
+                "LOG_LEVEL": "INFO",
+                "VERBOSE": True,
+                "CASE_DIR": None,
+                "CASEID": None,
+                "CASESTR": "historical",
+                "CALENDAR": "standard",
```

Comment:
I think this value need to match the value of the calendar in the VIC-GL output file. I just looked at some files for the Peace River, and they all had the "standard" calendar, even though I expected some of them to have different calendars. I've asked Arelia whether VIC-GL standardizes calendars on all its inputs.

If not, we'll need a more flexible way to handle the calendar sooner or later.

Reply:
Arelia says hydrology converts all model outpus to a standard calendar before inputting it to VIC-GL. So we only need to support a standard calendar here, which is excellent.

Reply:
Multiple calendars are a giant headache.